config/locales/activerecord.nl.yml

```diff
@@ -135,7 +135,8 @@ nl:
         title: Titel
         updated_at: Bijgewerkt op
         more_info_flag: Toon in extra informatie pagina
-        print_content_flag: Print inhoud 
+        print_content_flag: Print inhoud
+        locale: Taal
```

Comment:
👏  for the extra effort in translations! (BTW: We're removing from specs the need to add translations for any language but English, new strings will be translated at https://crowdin.com/project/consul/)